apps/lockfile-explorer/src/graph/test/graphTestHelpers.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import yaml from 'js-yaml';
+import { FileSystem, NewlineKind } from '@rushstack/node-core-library';
+import {
+  type IJsonLfxGraph,
+  type IJsonLfxWorkspace,
+  lfxGraphSerializer,
+  type LfxGraph
+} from '../../../temp/lfx-shared';
+
+import * as lfxGraphLoader from '../lfxGraphLoader';
+
+const FIXTURES_FOLDER: string = path.resolve(__dirname, '../../../src/graph/test/fixtures/');
+
+export async function loadAndSerializeLFxGraph(options: {
```

Comment:
```suggestion
export async function loadAndSerializeLFxGraphAsync(options: {
```

Reply:
I'm surprised a lint rule didn't complain about this.

Reply:
Fixed